snapshot/snapshot.go

```diff
@@ -334,6 +345,15 @@ func (o *snapshotter) Prepare(ctx context.Context, key, parent string, opts ...s
 		return nil, err
 	}
 
+	metricBeginTime := time.Now()
+	defer func(id *string) {
+		metricTime := time.Since(metricBeginTime)
+
+		if err := exporter.ExportPrepareTimeMetric(*id, metricTime.Seconds()); err != nil {
```

Comment:
It's better to record the snapshot key or ID as a record label so we can analyze how long contained spans calling two APIes

Reply:
Yes,  it records the snapshot key now.